Dockerfile

```diff
@@ -1,4 +1,14 @@
 FROM ruby:2.6.6-slim-buster
 COPY app /app
+
+RUN apt-get update -yqq && apt-get -yqq --no-install-recommends install build-essential
+
+COPY Gemfile Gemfile
+COPY Gemfile.lock Gemfile.lock
+RUN gem install bundler
+RUN bundle config set with 'development test'
```

Comment:
Why do we need development test here? When the code runs inside the image, it should only need production dependencies IMO

Reply:
You're right